lib/ReactJsonSchema.js

```diff
@@ -18,19 +18,25 @@ export default class ReactJsonSchema {
   parseSubSchemas(subSchemas = []) {
     const Components = [];
     let index = 0;
-    for (const subSchema of subSchemas) {
+    Object.keys(subSchemas).forEach((key) => {
```

Comment:
Any browser that implement `Object.keys` also implements `Array.isArray`.

Reply:
@wtgtybhertgeghgtwtg 

Yeah you're right. We ran into an issue with Array.isArray internally here with another project so I was just trying to be vigilant in this project. We've decided to switch back to using Array.isArray since that's what react supports. If there's an issue, people can shim the same way they would with react: https://reactjs.org/docs/react-dom.html#browser-support. 

Thanks again for the attention